layers/+lang/scheme/funcs.el

```diff
@@ -0,0 +1,4 @@
+(defun geiser-customize-implementation ()
```

Comment:
what's this for?

Reply:
Ah well, as up to now I only use very few geiser functions, I just used the `geiser` menu in the menu-bar to check out the available geiser functions in the different geiser modes.
This menu has an entry to jump to the customize menu for the currently used scheme implementation.
I thought that is might be a nice service, to provide a similar function under the `SPC m h` menu, but I am just trying it out.
I am happy to hear any positive or negative opinion about it, maybe I am becoming too user friendly :laughing:

Reply:
I think this is not really useful.
If you find particular customization variables that are commonly used on a per-file/per-project bases, you can make some toggle functions and bind them under `SPC m T`.

That provides easy access to particular, concrete functionalities.